python/taichi/shaders/Postprocessing_vk.vert

```diff
@@ -0,0 +1,14 @@
+#version 450
+
+vec2 positions[] = {
+  vec2(-1.0, 4.0),
+  vec2(-1.0, -1.0),
+  vec2(4.0, -1.0)
+};
+
+layout(location = 0) out vec2 uv;
+
+void main() {
+  uv = positions[gl_VertexIndex] * 0.5 + 0.5;
```

Comment:
If you use `gl_VertexIndex`, it will be harder for us to move GGUI to OpenGL once we have OpenGLDevice. AFAIK opengl doesn't support `gl_VertexIndex`

Reply:
In OpenGL there is gl_VertexID and it is fully equivalent, so we don't need to worry about that.